client/src/core/client/stream/tabs/Notifications/NotificationsContainer.tsx

```diff
@@ -44,6 +45,36 @@ const NotificationsContainer: FunctionComponent<Props> = ({
       <div className={styles.title}>
         <Localized id="notifications-title">Notifications</Localized>
       </div>
+      <div className={styles.methodOfRedress}>
+        {settings.dsa.methodOfRedress.method ===
+          GQLDSA_METHOD_OF_REDRESS.NONE && (
+          <Localized id="notifications-methodOfRedress-none">
+            All moderation decisions are final and cannot be appealed
+          </Localized>
+        )}
+        {settings.dsa.methodOfRedress.method ===
+          GQLDSA_METHOD_OF_REDRESS.EMAIL && (
+          <Localized
+            id="notifications-methodOfRedress-email"
+            vars={{
+              email: settings.dsa.methodOfRedress.email,
+            }}
+          >
+            {`To appeal a decision that appears here please contact ${settings.dsa.methodOfRedress.email}`}
+          </Localized>
+        )}
+        {settings.dsa.methodOfRedress.method ===
+          GQLDSA_METHOD_OF_REDRESS.URL && (
+          <Localized
+            id="notifications-methodOfRedress-url"
+            vars={{
+              url: settings.dsa.methodOfRedress.url,
+            }}
+          >
+            {`To appeal a decision that appears here please visit ${settings.dsa.methodOfRedress.url}`}
```

Comment:
Just curious, should the email and url here be links that are actionable? Or we just expect folks to copy and paste into their browser/email?